### PR TITLE
Normalize fxhash contract name in community pills

### PR DESCRIPTION
**Pull request by Robinnnnn**
### Summary of Changes

Converts FxHash family contract names to "FxHash". This is hack until we have proper sub-community support.

| Before | After |
|--------|--------|
| image | image | 

### Demo or Before/After Pics

**Before**
| Post                                    | Detail Page                                     |
| ------------------------------------------ | ------------------------------------------ |
| image | image |

**After**
| Post                                    | Detail Page                                     |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-03 at 11 42 24 PM | Screenshot 2023-10-03 at 11 42 19 PM |

**Mobile After**

| Post                                    | Detail Page                                     |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-03 at 11 47 02 PM | Screenshot 2023-10-03 at 11 49 03 PM |

### Edge Cases

- [x] Ensure other contracts don't regress

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [x] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

**Review by kaitoo1 — APPROVED** [review]

File: apps/mobile/src/screens/NftDetailScreen/NftDetailSection.tsx [hunk 1]
```diff
@@ -152,6 +153,10 @@ export function NftDetailSection({ onShare, queryRef }: Props) {
   //   }
   // }, [navigation, track, token.creator?.username]);
 
+  const contractName = isFxHashContractAddress(token.contract?.contractAddress?.address)
+    ? 'FxHash'
```
Comment (kaitoo1) [inline apps/mobile/src/screens/NftDetailScreen/NftDetailSection.tsx:157]: I think they prefer to have it stylized as `fxhash`
